unfetter-utils/user_promotion_tool.py

```diff
@@ -17,16 +17,8 @@
     client = MongoClient(MONGO_HOST, MONGO_PORT)
     db = client[UNFETTER_DATABASE]
     users_collection = db[UNFETTER_USERS_COLLECTION]
-    users_raw = users_collection.find({})
+    users_raw = users_collection.find({ 'registered': True })
     users = [user for user in users_raw]
```

Comment:
I believe that this was left in by mistake.  removed since new line 32 is printing out the users.